src/test/java/com/google/step/youtube/YouTubeInfoScraperTest.java

```diff
@@ -52,133 +45,80 @@
     private static final String USERNAME = "USERNAME";
     private static final String UPLOAD_ID = "UPLOAD_ID";
     private static final String VIDEO_ID = "VIDEO_ID";
-    private static final String VIDEO_TITLE = "VIDEO_TITLE";
-    private static final String KEYWORD = "KEYWORD";
     private static final String NO_RESULTS_KEYWORD = "NO_RESULTS_KEYWORD";
-    private static final List<String> EMPTY_VIDEO_ID_LIST = Arrays.asList();
-    private static final List<String> VIDEO_ID_LIST = Arrays.asList();
-    private static final Date DATE = new Date(0L);
+    private static final List<String> EMPTY_VIDEO_ID_LIST = Collections.emptyList();
+    private static final List<String> VIDEO_ID_LIST = Collections.emptyList();
```

Comment:
are you planning on adding to this list?   If this is a mutable collection it is not a constant and should not be in all caps.  (in which case it might be mutable static state, which is something we want to avoid.). if it's immutable use ImmutableList.of()